federation/plugin/pkg/admission/schedulingpolicy/admission_test.go

```diff
@@ -343,6 +343,7 @@ func newControllerWithTestServer(f func(w http.ResponseWriter, r *http.Request),
 	if err != nil {
 		return nil, err
 	}
+	defer file.Close()
```

Comment:
@FengyunPan out of curiosity, what kind of accidents? In theory, I agree with what you are saying. But in this particular case, does it apply or matter?